telegram/_bot.py

```diff
@@ -372,7 +374,10 @@ async def initialize(self) -> None:
             return
 
         await asyncio.gather(self._request[0].initialize(), self._request[1].initialize())
-        await self.get_me()
+        try:
+            await self.get_me()
+        except InvalidToken as exc:
+            raise InvalidToken(f"The token {self.token} was rejected by the server.") from exc
```

Comment:
```suggestion
        try:
            await self.get_me()
        except InvalidToken as exc:
            raise InvalidToken(f"The token `{self.token}` was rejected by the server.") from exc
```

I guess a CSI-style comment on why we do this (at all, here and nowhere else) might be good